pkg/api/admin/api/config.go

```diff
@@ -122,9 +120,19 @@ type CertificateConfig struct {
 	// geneva integration certificates
 	GenevaLogging *Certificate `json:"genevaLogging,omitempty"`
 	GenevaMetrics *Certificate `json:"genevaMetrics,omitempty"`
+
+	// external provided certificates which are web facing
+	// must contain intermediate certificates too
+	// TODO: Move all certificates to be slice
+	OpenShiftConsole *CertificateChain `json:"-"`
+	Router           *CertificateChain `json:"-"`
 }
 
 // Certificate is an x509 certificate.
 type Certificate struct {
 	Cert *x509.Certificate `json:"cert,omitempty"`
 }
+
+type CertificateChain struct {
```

Comment:
nit - naming: Chain implies there's a cryptographic relationship between the objects in the chain. CertificateSlice? CertificateBundle?

Reply:
There should be a relationship. So I think its ok :)